app/assets/javascripts/reducers/index.coffee

```diff
@@ -36,11 +36,12 @@ Screensmart.reducer = Redux.combineReducers
         merge response,
               domain_ids: action.domainIds
       when 'SET_ANSWER'
-        responseWithoutNonFilledOutQuestions \
-          responseWithAnswer \
-            response,
-            action.id,
-            action.value
+        merge responseWithDoneFalse \
+                responseWithoutNonFilledOutQuestions \
```

Comment:
`responseWithoutNonFilledOutQuestions` bevat een dubbele ontkenning. Is het een `responseWithOnlyFilledOutQuestions`?